docker-compose.yml

```
@@ -21,4 +21,13 @@ services:
     ports:
       - "27017:27017"
     volumes:
-      - ./mongo_data:/data/db
\ No newline at end of file
+      - ./mongo_data:/data/db
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)